root/etc/cron.weekly/evebox-geoip

```diff
@@ -20,9 +20,8 @@
 # along with NethServer.  If not, see COPYING.
 #
 
-# Downalod GeoIP db for EveBox
+# Download GeoIP db for EveBox
 
-db=/etc/evebox/GeoLite2-City.mmdb.gz
+db=/usr/share/GeoIP/GeoLite2-City.mmdb
 rm -f $db
-curl -sL http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz -o $db
-chown suricata:suricata $db
+curl -sL http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz -o - | gunzip -N > $db
```

Comment:
(to verify) if the remote download fails, the `$db` is truncated. Maybe a more robust approach is performing the download to a temporary file, uncompress it, set permissions and if everything succeeds, moving it to the final location.

Reply:
>Maybe a more robust approach is performing the download to a temporary file, uncompress it, set permissions and if everything succeeds, moving it to the final location.

Agree, I need look in the check (exit codes of curl)  for verifying a successful download to accomplish this .

>set permissions
al my tests show it can be owned by root as long read permissions are set (everybody has read permissions by default here) , no need to chown it to suricata.

Reply:
Maybe if gunzip succedes you can be sure that also curl was fine.